### PR TITLE
amend preexisting conditions as per curators feedback

### LINKED ISSUE #619: Generate acronyms for preexisting conditions
So that the suggest box can suggest using those acronyms.

### PR DESCRIPTION
**Pull request by attwad**
Lauren and a bunch of interns went through the file and fixed/added/removed acronyms.

I'm glad they did because when I see the diffs I have no idea that "Nasu-Hakola disease" should be "Nasu-Hakola disease or polycystic lipomembranous osteodysplasia with sclerosing leukoencephalopathy (PLOSL)" really 🤷 

Fixes #619

### CODE REVIEW

File: suggest/preexisting_conditions.txt [hunk 1]
```diff
@@ -263,47 +263,47 @@ Charcot-Marie-Tooth disease recessive intermediate A (DRI)
 Charcot-Marie-Tooth disease recessive intermediate B (DRI)
```
Comment (allysonjp715) [inline suggest/preexisting_conditions.txt:263]: I would think these should be CMT* too

File: suggest/preexisting_conditions.txt [hunk 2]
```diff
@@ -2288,14 +2260,13 @@ campomelic dysplasia (CD)
 camptodactyly-arthropathy-coxa vara-pericarditis syndrome
 camptodactyly-tall stature-scoliosis-hearing loss syndrome (LS)
 campylobacteriosis
-cancer
+cancer (CA)
```
Comment (allysonjp715) [inline suggest/preexisting_conditions.txt:2263]: this one seems odd... but maybe I don't have the right background knowledge

File: suggest/preexisting_conditions.txt [hunk 3]
```diff
@@ -4229,12 +4156,12 @@ growth hormone secreting pituitary adenoma (GHSPA)
...
 gum cancer (GC)
 gummatous syphilis (GS)
-gynatresia
+gynatresia or vaginal stenosis
 gynecomastia
 gyrate atrophy (GA)
 haemonchiasis
 haemophilus meningitis (HM)
-hair disease (HD)
+hair disease (HD) or balding
```
Comment (allysonjp715) [inline suggest/preexisting_conditions.txt:4164]: I'm surprised HD should stay for this one

**Comment by allysonjp715** [comment]
> I'm glad they did because when I see the diffs I have no idea that "Nasu-Hakola disease" should be "Nasu-Hakola disease or polycystic lipomembranous osteodysplasia with sclerosing leukoencephalopathy (PLOSL)" really 🤷

😮

File: suggest/preexisting_conditions.txt (hunk 2 above)
Comment (attwad) [reply suggest/preexisting_conditions.txt:2263]: yeah removed.

File: suggest/preexisting_conditions.txt (hunk 3 above)
Comment (attwad) [reply suggest/preexisting_conditions.txt:4164]: removed.

File: suggest/preexisting_conditions.txt (hunk 1 above)
Comment (attwad) [reply suggest/preexisting_conditions.txt:263]: done.

**Comment by attwad** [comment]
sorted everything again, thanks

**Pull request merged**